packages/app-desktop/gui/NoteEditor/utils/contextMenu.ts

```diff
@@ -55,6 +22,12 @@ function handleCopyToClipboard(options: ContextMenuOptions) {
 	}
 }
 
+async function saveFileData(data: any, filename: string) {
+	const newFilePath = await bridge().showSaveDialog({ defaultPath: filename });
+	if (!newFilePath) return;
+	await fs.outputFile(newFilePath, data);
```

Comment:
Please use `writeFile` for consistency. Also import `writeFile` directly - we try not to import the whole fs namespace anymore, because that would make tree shaking more difficult (if we ever implement this). You don't need to update the other fs calls in this file.

Reply:
If we don't update the other calls to fs then I had to import like this
```
const fs = require('fs-extra');
const { writeFile } = require('fs-extra');
```
Is it alright?

Reply:
Yes sure, that's fine.